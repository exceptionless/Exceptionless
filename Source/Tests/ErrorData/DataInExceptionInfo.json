{
  "ExtendedData": {
    "User": "\"You\"",
  },
  "Inner": {
    "ExtendedData": {
      "__ExceptionInfo": "{\"Data\": {\"Blah\": \"Blah\"}}"
    }
  }
}