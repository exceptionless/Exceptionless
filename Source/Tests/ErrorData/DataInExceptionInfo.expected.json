{
  "Data": {
    "User": "\"You\""
  },
  "Type": "error",
  "@error": {
    "Inner": {
      "Data": {
        "@ext": {
          "Data": {
            "Blah": "Blah"
          }
        }
      }
    }
  }
}